documentation/chatmeleon-swagger.yaml

```diff
@@ -161,37 +162,40 @@ paths:
                     type: string
                     pattern: "^[0-9a-f]{24}$"
                     description: The ID of the earliest message of the current messages page.
+        "401":
+          description: Unauthorized - JWT token is missing or invalid.
+        "403":
+          description: Forbidden - User not allowed to retrieve the messages of the conversation.
```

Comment:
When this happen?

Reply:
Again, someone should be able to retrieve the message when they are part of the group chat that contains the message or if they are the actual sender of the message.  I was just being more careful with a possible breach of security, where they could be retrieving random messages where they are not a part of

Reply:
Defo. Just note my above comment

Reply:
Same thing above